docs/iris/src/userguide/loading_iris_cubes.rst

```diff
@@ -314,21 +315,53 @@ for many years:
 
 .. doctest:: timeseries_range
     :options: +NORMALIZE_WHITESPACE, +ELLIPSIS
-    
+
     >>> print(long_ts.coord('time'))
     DimCoord([2007-04-09 00:00:00, 2007-04-16 00:00:00, 2007-04-23 00:00:00,
               ...
               2010-02-01 00:00:00, 2010-02-08 00:00:00, 2010-02-15 00:00:00],
              standard_name='time', calendar='gregorian')
 
-We can select points within a certain part of the year, in this case between
-the 15th of July through to the 25th of August, by combining the datetime cell
-functionality with PartialDateTime:
+Given two dates in datetime format, we can select all points between them.
+
+.. doctest:: timeseries_range
+    :options: +NORMALIZE_WHITESPACE, +ELLIPSIS
+
+    >>> d1 = datetime.datetime.strptime('20070715T0000Z', '%Y%m%dT%H%MZ')
+    >>> d2 = datetime.datetime.strptime('20070825T0000Z', '%Y%m%dT%H%MZ')
+    >>> st_swithuns_daterange_07 = iris.Constraint(
+    ...     time=lambda cell: d1 <= cell.point < d2)
+    >>> within_st_swithuns_07 = long_ts.extract(st_swithuns_daterange_07)
+    >>> print(within_st_swithuns_07.coord('time'))
+    DimCoord([2007-07-16 00:00:00, 2007-07-23 00:00:00, 2007-07-30 00:00:00,
+              2007-08-06 00:00:00, 2007-08-13 00:00:00, 2007-08-20 00:00:00],
+             standard_name='time', calendar='gregorian')
+
+Alternatively, we may do the same thing with :class:`iris.time.PartialDateTime`
+objects.
```

Comment:
> Alternatively, we may do the same thing with ...

This phrasing feels a bit too informal.
Could you rephrase to be something like:
> Alternatively, we may rewrite this using PartialDateTime objects ...